src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected, displayTab }) => {
+  return (
```

Comment:
Add the same declaration for displayTab here. Just copy-paste the line you have at `App`, before this return, and remove the prop with the same name